filament/src/details/ColorGrading.cpp

```diff
@@ -658,27 +660,28 @@ FColorGrading::FColorGrading(FEngine& engine, const Builder& builder) {
 
     DriverApi& driver = engine.getDriverApi();
 
-    Config c;
-    // This lock protects the data inside Config, which is written to by the Filament thread,
-    // and read from multiple Job threads.
-    Mutex configLock;
-    {
-        std::lock_guard<Mutex> const lock(configLock);
-        c.lutDimension          = builder->dimension;
-        c.adaptationTransform   = adaptationTransform(builder->whiteBalance);
-        c.colorGradingIn        = selectColorGradingTransformIn(builder->toneMapping);
-        c.colorGradingOut       = selectColorGradingTransformOut(builder->toneMapping);
-        c.colorGradingLuminance = selectColorGradingLuminance(builder->toneMapping);
-        c.oetf                  = selectOETF(builder->outputColorSpace);
-    }
+    const Config config = {
```

Comment:
so I see you removed the shenanigan with the Mutex. I agree with you that it shouldn't be needed. But I think ASAN was complaining and/or we ran into problems, and we never understood what it was. So for now, I would suggest to keep the code the way it was and we can make that specific change in a separate PR.

Reply:
Is it not resolvable by having config be declared const? The issue seemed to me that ASAN thought multiple threads might be writing to ColorGrading.cpp. I guess it's not that simple...?

I'll ask offline more about this.